frontend/src/app/versions.tsx

```diff
@@ -372,18 +376,21 @@ class Versions extends Component<IProps, any> {
                             const moduleVersion = responseJSON["__children__"][i]
                             if (moduleVersion["jcr:uuid"] === draftTag) {
                                 this.draft[0]["version"] = "Version " + moduleVersion["__name__"];
-                                this.draft[0]["updatedDate"] = moduleVersion["jcr:lastModified"];
-                                this.draft[0]["metaData"] = this.getHarrayChildNamed(moduleVersion, "metadata")
-                                this.draft[0]["path"] = "/content/" + this.props.modulePath + "/en_US/" + moduleVersion["__name__"];
-                                this.props.draftUpdateDate(this.draft[0]["updatedDate"], "draft", this.draft[0]["path"]);
+                                this.draft[0]["metadata"] = this.getHarrayChildNamed(moduleVersion, "metadata")
+                                this.draft[0]["updatedDate"] = this.draft[0]["metadata"]["pant:dateUploaded"] !== undefined ? this.draft[0]["metadata"]["pant:dateUploaded"] : '';
```

Comment:
This is weird... why do we need to copy the property `draft.metadata.pant:dateUploaded` into `draft.updatedDate`? Can't the consumer of this data just use `draft.metadata.pant:dateUploaded` directly?